docs/Cloud_integrations/Device_Integration_Center/webhooks.md

```diff
@@ -60,17 +56,16 @@ Webhook parameters include:
 
 To validate the webhook parameters, it is possible to **test the event handler** before creating the event handler.
 
-![Test event handler](images/test-event-handler.png "Test event handler")
+![Test event handler](images/test-event-handler.webp "Test event handler")
 
 !!! Tip "Influx and Datacake"
-    For instructions on integrating data with **Influx**, visit the [InfluxDB guide](influx.md). To see how to integrate with **Datacake**, see the [Datacake guide](datacake.md).
-
-
-![Define webhook parameters](images/webhook-parameters.png "Define webhook parameters")
+    For instructions on integrating data with **Influx**, visit the [InfluxDB guide](integrations/influx.md). To see how to integrate with **Datacake**, see the [Datacake guide](integrations/datacake.md).
+ 
+![Define webhook parameters](images/webhook-parameters.webp "Define webhook parameters")
 
 Click **Next step** and **Add event handler** to create the event handler.
 
-![Active webhook](images/active-webhook.png "Active webhook")
+![Active webhook](images/active-webhook.webp "Active webhook")
 
 !!! warning
     An event handler is automatically deactivated after several consecutive failed attempts. Manually reactivate the event handler to continue using the data integration.
```

Comment:
You have to  manually reactivate the event handler to continue using the data integration.